plugins/nanospell/plugin.js

```diff
@@ -564,13 +580,17 @@
 				return editor.getSelection().getSelectedText().length == 0;
 			}
 
-			var spellTicker = null;
+			function startSpellCheckTimer(delay) {
+				if (self._timer) {
```

Comment:
I think I mostly did this to be able to use self._timer = 0 to clear a completed timeout id between runs.